src/intentManager.js

```diff
@@ -319,7 +324,10 @@ define([
 
     $.vellum.plugin("intents", {}, {
         init: function() {
-            refreshCurrentMug = this.refreshCurrentMug.bind(this);
```

Comment:
It actually hasn't been needed since I pulled out field auto completion for the printing question (custom feature and had some issues with mediauploader)